logic/src/commonMain/kotlin/com/wire/kalium/logic/data/asset/Asset.kt

```diff
@@ -28,4 +28,4 @@ enum class RetentionType {
     EXPIRING
 }
 
-fun isValidImage(mimeType: String): Boolean = mimeType in setOf("image/jpg", "image/jpeg", "image/png", "image/heic")
+fun isValidImage(mimeType: String): Boolean = mimeType in setOf("image/jpg", "image/jpeg", "image/png", "image/heic", "image/gif")
```

Comment:
Small suggestion about giving this function a meaningful name, and maybe this should be platform-dependent (aka move it to AR)?
```suggestion
fun supportedInlieImages(mimeType: String): Boolean = mimeType in setOf("image/jpg", "image/jpeg", "image/png", "image/heic", "image/gif")
```

Reply:
What about `isDisplayableMimeType` ?